customer/views.py

```diff
@@ -17,7 +19,14 @@ class SignUp(FormView):
     success_url = "/customer/thanks/"
 
     def form_valid(self, form):
-        print(f"FORM DATA: {form.cleaned_data}")
+        repository = DjangoCustomerRepository()
```

Comment:
This should be replaced with the orchestration layer defined in `service_layer.customer`.